run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cd ..
+echo 'Running tests...'
+
+EXCLUDE_TESTS=HTTPDataStoreCSVBenchmarks,HTTPDataStoreBenchmarks,MicroBenchmarks,TDBDataStoreBenchmarks,FakeDataStoreBenchmarks,AverageMicroBenchmarks,SqliteDataStoreBenchmarks,DataStoreCSVBenchmarks,TDBDataStoreCSVBenchmarks,AFF4Benchmark
```

Comment:
Dont the benchmark tests get excluded anyway when running tests with the small label?